src/shared/services/configuration/endpoint-definitions.js

```diff
@@ -94,7 +94,7 @@ export default {
     '/signals/v1/private/status-messages/category/',
   // Private. Used by the incident management module to retrieve and query standard texts.
   STANDARD_TEXTS_ENDPOINT: '/signals/v1/private/status-messages/',
-  // Private. Used by the incident management module to retrieve and search standard texts.
+  // Private. Used by the incident management module to retrieve and search standard texts, to patch updated data or to delete a standard text..
```

Comment:
![Uploading Screenshot 2023-07-04 at 10.59.43.png…]()

Reply:
this is just a get

Reply:
I use this endpoint now also to patch and delete.